lib/core/class-groups-user.php

```diff
@@ -128,6 +128,16 @@ public function __get( $name ) {
 
 		if ( $this->user !== null ) {
 
+			/**
+			 * Filter the user properties.
+			 * @param WP_User
+			 * @param string
+			 */
+			$null = apply_filters( 'groups_user_get_' . $name, null, $this->user, $name );
```

Comment:
reversed approach - should be applied on result instead (before returned)

Reply:
This method is essentially a shortcut which will prevent any unnecessary logic to run and take up resources.
The same method is used in WP core many time.